cypress/test/newsroom/NewsroomBlogPost.cy.js

```diff
@@ -0,0 +1,29 @@
+import { mount } from 'cypress/react'
+import NewsroomBlogPosts from '../../../components/newsroom/NewsroomBlogPosts';
+
+
+describe('Features Component', () => {
+    it('renders without errors', () => {
+        mount(<NewsroomBlogPosts />);
+
+    });
+    it('checks if swiper component is present', () => {
+        mount(<NewsroomBlogPosts />);
+        cy.get('[data-testid="NewsroomBlog-main-div"]').should('exist')
+
+    });
+    it('slides to the next slide', () => {
+        mount(<NewsroomBlogPosts />);
+        cy.get('[data-testid="Blog-Next"]').click();
```

Comment:
Make the click event work automatically using `trigger` or `{ force: true }`.

Applicable everywhere.

Reply:
done